src/bid/bid.rs

```diff
@@ -24,21 +24,13 @@ pub struct Bid {
     pub(crate) elegibility_ts: u32,
     // t_e
     pub(crate) expiration_ts: u32,
-    //
-    // Public Outputs
-    //
     // i (One time identity of the prover)
     pub(crate) prover_id: BlsScalar,
     // q (Score of the bid)
     pub(crate) score: Score,
-    // b (blinder)
-    pub(crate) blinder: JubJubScalar,
-    // b_enc (encrypted blinder) // XXX: Scalar for now. Double check
-    pub(crate) encrypted_blinder: JubJubScalar,
-    // v (Bid value)
-    pub(crate) value: JubJubScalar,
-    // v_enc (encrypted_value)
-    pub(crate) encrypted_value: JubJubScalar,
+    // The encrypted value & blinder fields were previously encrypted with `ElGamal` schema.
+    pub(crate) encrypted_blinder: (AffinePoint, AffinePoint),
```

Comment:
Its unclear to me why the encrypted blinder is represented with 2 points. If its a poseidon cipher, should be either 3 scalars `Fq` or `PoseidonCipher` itself

Reply:
As mentioned @toghrulmaharramov yesterday the encrypted values with ElGamal will be represented by to AffinePoints.

Reply:
Ok, but later on we may replace this for the encrypted structure to benefit from the homomorphic properties of elgamal cipher